score/ipython/Score_Dissolve_Script.ipynb

```diff
@@ -0,0 +1,459 @@
+        "    return state_tracts\n",
+        "\n",
+        "\n",
+        "def create_buckets_from_tracts(state_tracts: pd.DataFrame):\n",
+        "    # assign tracts to buckets by D_SCORE\n",
+        "    state_tracts.sort_values(\"D_SCORE\", inplace=True)\n",
+        "    D_SCORE_bucket = []\n",
+        "    num_buckets = 10\n",
+        "    bucket_size = math.ceil(len(state_tracts.index) / num_buckets)\n",
+        "    for i in range(len(state_tracts.index)):\n",
+        "        D_SCORE_bucket.extend([math.floor(i / bucket_size)])\n",
+        "    state_tracts[\"D_SCORE_bucket\"] = D_SCORE_bucket\n",
+        "    return state_tracts\n",
+        "\n",
+        "\n",
+        "def aggregate_buckets(state_tracts: pd.DataFrame):\n",
+        "    # dissolve tracts by bucket\n",
+        "    state_attr = state_tracts[[\"D_SCORE\", \"D_SCORE_bucket\", \"geometry\"]].reset_index(\n",
+        "        drop=True\n",
+        "    )\n",
+        "    state_dissolve = state_attr.dissolve(by=\"D_SCORE_bucket\", aggfunc=\"mean\")\n",
+        "    return state_dissolve\n",
+        "\n",
+        "\n",
+        "def breakup_multipolygons(state_bucketed_df: pd.DataFrame, num_buckets: int):\n",
+        "    compressed = []\n",
+        "    for i in range(num_buckets):\n",
+        "        for j in range(len(state_bucketed_df[\"geometry\"][i].geoms)):\n",
+        "            compressed.append(\n",
+        "                [\n",
+        "                    state_bucketed_df[\"D_SCORE\"][i],\n",
+        "                    state_bucketed_df[\"geometry\"][i].geoms[j],\n",
+        "                ]\n",
+        "            )\n",
+        "    return compressed\n",
+        "\n",
+        "\n",
+        "def write_to_file(compressed: pd.DataFrame, file_name: str):\n",
+        "    gdf_compressed = gpd.GeoDataFrame(\n",
+        "        compressed, columns=[\"D_SCORE\", \"geometry\"], crs=\"EPSG:4326\"\n",
+        "    )\n",
+        "    gdf_compressed.to_file(f\"{file_name}_low.geojson\", driver=\"GeoJSON\")\n",
+        "\n",
+        "\n",
+        "def process_file(file_name: str, usa_df: pd.DataFrame):\n",
+        "    print(f\"Processing file {file_name}...\")\n",
+        "    state_merged_simplified = merge_and_simplify_file(file_name, usa_df)\n",
+        "    state_tracts = aggregate_to_tracts(state_merged_simplified)\n",
+        "    state_tracts = create_buckets_from_tracts(state_tracts)\n",
+        "    state_bucketed_df = aggregate_buckets(state_tracts)\n",
+        "    compressed = breakup_multipolygons(state_bucketed_df, 10)\n",
+        "    write_to_file(compressed, file_name)"
+      ],
+      "outputs": [],
+      "metadata": {
+        "id": "UEemuUJNj-kY"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "source": [
+        "score_df = pd.read_csv(\"/content/usa.csv\", dtype={\"GEOID10\": \"object\"})"
```

Comment:
This returns an error -- it would be preferable to use data source helpers like the ones we used in [previous notebooks](https://github.com/usds/justice40-tool/blob/issue-271-comparison-tool-refactor/score/ipython/census_etl.ipynb):

```

import pandas as pd
import censusdata
import csv
from pathlib import Path
import os
import sys

module_path = os.path.abspath(os.path.join(".."))
if module_path not in sys.path:
    sys.path.append(module_path)

from etl.sources.census.etl_utils import get_state_fips_codes

ACS_YEAR = 2019

DATA_PATH = Path.cwd().parent / "data"
```